examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -137,11 +138,21 @@ def main():
         default=8,
         help="how many samples to produce for each given prompt. A.k.a. batch size",
     )
+    parser.add_argument(
+        "--ddim",
+        action='store_true',
+        help="use ddim sampling",
+    )
     parser.add_argument(
         "--dpm_solver",
         action='store_true',
         help="use dpm_solver sampling",
     )
+    parser.add_argument(
+        "--dpm_solver_pp",
```

Comment:
The default value for `ddim_steps` is 50, which may not be appropriate for dpm_solver_pp  (15?). It is recommended to illustrate the recommended sampling steps in for users. e.g., in help info, or raise warning if the sampling step is not appropriate for the selected samplers.